pvacseq/lib/main.py

```diff
@@ -143,6 +149,13 @@ def main(args_input = sys.argv[1:]):
     if args.fasta_size%2 != 0:
         sys.exit("The fasta size needs to be an even number")
 
+    if args.downstream_sequence_length == 'full':
+        downstream_sequence_length = None
+    elif args.downstream_sequence_length.isdigit():
+        downstream_sequence_length = args.downstream_sequence_length
+    else:
+        sys.exit("The downstream sequence length needs to be a positive integer or 'full'")
```

Comment:
Does anything bad happen if this ends up being `0`?  This line suggests it should be positive, but `isdigit()` allows `0`.